client/me/purchases/manage-purchase/notices.tsx

```diff
@@ -100,20 +100,41 @@ class PurchaseNotice extends Component<
 			return this.getExpiringLaterText( purchase );
 		}
 
-		if ( isMonthly( purchase.productSlug ) ) {
-			const daysToExpiry = moment( expiry.diff( moment() ) ).format( 'D' );
+		if ( isMonthlyPurchase( purchase ) ) {
+			const daysToExpiry = expiry.diff( moment(), 'days' );
+
+			if ( isTemporarySitePurchase( purchase ) ) {
+				return translate(
+					'%(purchaseName)s will expire and be removed in %(daysToExpiry)d days. ',
```

Comment:
Back to my original minor comment: there's an extra space here.

```suggestion
					'%(purchaseName)s will expire and be removed in %(daysToExpiry)d days.',
```